partner_identification/tests/test_res_partner.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import common
+from odoo.exceptions import ValidationError
+from .fake_models import ResPartner, setup_test_model, teardown_test_model
+
+
+class TestResPartner(common.SavepointCase):
+
+    at_install = False
```

Comment:
@tarteo PLZ remove `at_install` and `pot_install`

Reply:
Thanks, removed them.